wd_demo.au3

```diff
@@ -240,6 +241,9 @@ Func _RunDemo_Update($idUpdate, $sBrowserName)
 
 	Local $bUpdateResult = _WD_UpdateDriver($sBrowserName, @ScriptDir, $bFlag64, $bForce)
 	ConsoleWrite('> UpdateResult = ' & $bUpdateResult & @CRLF)
+	Local $iErr = @error, $iExt = @extended
+	ConsoleWrite('> UpdateResult = ' & $bUpdateResult & @CRLF)
```

Comment:
This looks like a duplicate

Reply:
Lines 243 & 245 that is

Reply:
should be fixed with: [fix](https://github.com/Danp2/au3WebDriver/pull/507/commits/1f1913346349dc417bd82a621ad295ed70546c4f)